src/main/java/org/mifos/connector/airtel/mockairtel/utils/TransactionStatus.java

```diff
@@ -0,0 +1,5 @@
+package org.mifos.connector.airtel.mockairtel.utils;
+
+public enum TransactionStatus {
```

Comment:
Why this enum is required? You can use TransferStatus enum here.

Reply:
Done.